.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Run watir-rails CI
+
+'on':
+  pull_request:
```

Comment:
I'm not too familiar with GH Actions, but is this empty `pull_request` property needed here?

Reply:
It runs workflow on `pull_request` event with [default activity types](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request) (opened, synchronize, or reopened).